css/style.css

```diff
@@ -23,35 +31,81 @@ h1 {
 .main {
   border: 2px solid #085311;
   height: 720px;
-  width: 370px;
+  margin: 0px 15px;
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
+/*cards and content*/
 .card {
   display: flex;
   flex-direction: column;
   align-items: center;
   border: 2px solid #53d562;
   box-shadow: 0px 0px 7px 2px;
-  margin: 25px 15px 5px;
-  width: 300px;
+  margin: 25px 0px;
+  width: 84%;
   height: 200px;
```

Comment:
We don't recommend to set fix values for `height` because the elements take the space they need automatically. If the children need more space than 200px, the content will go beyond the `.card`. Either remove it or try `height: auto`.